gulpfile.mjs

```diff
@@ -684,6 +684,9 @@ function createTestSource(testsName, { bot = false, xfaOnly = false } = {}) {
     if (process.argv.includes("--noChrome") || forceNoChrome) {
       args.push("--noChrome");
     }
+    if (process.argv.includes("--headless")) {
+      args.push("--headless");
+    }
```

Comment:
How about adding the same code in the `makeRef` function below, for consistency, since I'm hoping that it's only a matter of time until we convert the rest of the botio commands?

Reply:
Good point; that one was indeed omitted by accident.